src/utils/theme.ts

```diff
@@ -15,9 +15,9 @@ setThemeVars('default', {
   '--gutter': 16,
 });
 
-export async function updateTheme() {
+export async function updateTheme(): Promise<boolean> {
   // Add device width and height
-  const { width, height } = Dimensions.get('window');
+  const { width, height } = Dimensions.get('window'); // tslint:disable-line no-backbone-get-set-outside-model
```

Comment:
I'm scared we will end up with few disable-line for this rule, we often use `Dimensions.get()` and probably others

Reply:
Yes, let's disable this plugin